src/test/resources/ocr-data/invalid/invalid-ocr.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
There's a test file named `duplicate-field-name.json` in the same directory. It's almost identical, so I think you could just use that one.

Reply:
done, deleted the new file and reusing the existing one